site/sfguides/src/tempo/tempo.md

```diff
@@ -0,0 +1,228 @@
+
+2. Download the CSV file from the TEMPO Snowflake app output.
+
+3. In Splunk, go to Settings > Add Data > Upload > select your CSV file.
+
+4. Follow the prompts to load the CSV, using default options.
+
+### Create the Dashboard
+Duration: 2
+
+1. After loading the CSV, click "Build Dashboards" > "Create New Dashboard".
+
+2. Select "Classic Dashboard Builder" and create the dashboard.
+
+3. In the dashboard editor, switch to "Source" view.
+
+4. Copy the XML from `anomaly_hub.xml` and paste it into the Source view.
+
+5. Save the Dashboard
+   - After the dashboard is saved, you will now have to create a Tempo Splunk macro
+
+7. Create a Tempo macro in Splunk
+   - In splunk create a new Splunk macro by going to ```Settings``` > ```Advanced Search``` > ```+ Add New```
+   - Keep Destination app as ```search```
+   - Name the macro ```TempoDataLocation```
+   - Define the macro as your Splunk path to Tempo's csv output. Will look something like this
+   ```xml
+   source="your-filename.csv" host="Your Name" sourcetype="csv"
+   ```
+   - You can leave the rest of the macro creation blank.
+   - Save the macro
+
+You should now be able to see the incidents - or anomalies - in your new dashboard.  This enables Security Operations teams to click through on the context provided by Tempo.  For example, you can see all transactions to and from a specific IP address, or across given ports, as a part of investigating the incidents that have been identified.
+
+Note that as a default, only the incidents are uploaded.  Not also transferring and loading the entire dataset of logs simplifies the work of the Security Operator and also can translate into significant cost savings, as Splunk and most security operations solutions tend to charge by data ingested.  
+
+### Important Notes
+- Set strong passwords for all user accounts.
+- Ensure sufficient disk space (minimum 10GB recommended).
+- Keep your EC2 instance and Splunk Enterprise updated with the latest security patches.
+
+### Troubleshooting
+- If the script fails to find the Splunk tarball, confirm that it's in the correct directory.
+- For user creation issues, check Splunk server logs and verify authentication credentials.
+
+<!-- ------------------------ -->
+## Conclusion And Resources
+Duration: 1
+
+### Conclusion
+
+Congratulations, you just ran the world's first purpose-built LogLM available as a Snowflake NativeApp.  In the weeks to come DeepTempo will launch a range of additional easy-to-use options and extensions as NativeApps, including tooling to simplify the process of using your own data with Tempo and upgrades to the power of Tempo including scale out multi-GPU usage. 
+
+### What You Learned
+- How to run Tempo on Snowflake as a NativeApp
+- How to monitor Tempo
+- How to see incidents and relevant context identified by Tempo in Splunk
+
+### Resources
+
+Please share your feedback, questions, and suggestions.  
```

Comment:
@mando222 Replace this sentence with links to resources. See https://quickstarts.snowflake.com/guide/getting_started_with_snowflake_arctic/index.html#5

Reply:
We aren't publicly posting additional resources outside of snowflake yet.  I understand what you are looking for but currently it doesn't exist.

Reply:
@mando222 In that case, can you add a link to https://www.snowflake.com/en/data-cloud/workloads/applications/native-apps/ and remove "Please share your feedback, questions, and suggestions."?

Reply:
done